src/interface/metadata.hpp

```diff
@@ -625,7 +622,9 @@ class Metadata {
 
  private:
   /// the attribute flags that are set for the class
-  refinement::RefinementFunctions_t refinement_funcs_;
+  refinement::RefinementFunctions_t refinement_funcs_ =
+      refinement::RefinementFunctions_t::RegisterOps<
+          refinement_ops::ProlongateCellMinMod, refinement_ops::RestrictCellAverage>();
```

Comment:
I could have left this in the constructor, but this felt more appropriate, it can still be re-set by the user.